squad/frontend/templates/squad/project.html

```diff
@@ -1,37 +1,51 @@
 {% extends "squad/base.html" %}
 {% load humanize %}
+{% load squad %}
 
 {% block content %}
-{% include "squad/project-nav.html" %}
 
 {% with status=project.status %}
-<table class='table table-striped'>
-  <tr>
-    <th>Last update</th>
-    <th>Tests</th>
-    <th>Metrics</th>
-  </tr>
-  <tr>
-    <td>
-      {{status.last_updated|default:"(no data)"}}
-      <em>{{status.last_updated|naturaltime}}</em>
-    </td>
-    <td>
-      {% if status.has_tests %}
-      <div class="progress" title="Pass: {{status.pass_percentage|stringformat:"d"}}%">
-        <div class="progress-bar progress-bar-success" style="width: {{status.pass_percentage}}%">
-          <span class="sr-only">Pass: {{status.pass_percentage}}% </span>
-        </div>
-        <div class="progress-bar progress-bar-danger" style="width: {{status.non_pass_percentage}}%">
-        </div>
-      </div>
-      {% else %}
-      (no data)
-      {% endif %}
-    </td>
-    <td>
-      {{status.metrics_summary|default:"(no data)"}}
-    </td>
-</table>
+<h1> <a href="{% group_url project.group %}">{{project.group.slug}}</a>/<a href="{% project_url project %}">{{project.slug}}</a></h1>
+<h3>Last build - <a href="{% project_url last_build %}">{{last_build.version}}</a> {{last_build.datetime}} {{last_build.datetime|naturaltime}}</h3>
+  <table class='table table-hover table-striped table-condensed'>
+    <tr>
+        <td colspan='4'><h3>Metadata</h3></td>
+    </tr>
+    {% for key, value in metadata %}
+    <tr>
+        <th>{{key}}</th>
+        <td>{{value|urlize}}<td/>
+    </tr>
+    {% endfor %}
+  </table>
+
+<h3>All builds</h3>
+  <table class='table table-striped'>
+    <tr>
+      <th>Build</th>
+      <th>Date</th>
+      <th># of Testjobs</th>
+      <th>Summary</th>
+    </tr>
+    {% for build in builds %}
+      <tr>
+        <td><a href="{% project_url build %}">{{build.version}}</a></td>
+        <td>
+          <a href="{% project_url build %}">{{build.datetime}}</a>
+          <em>{{build.datetime|naturaltime}}</em>
+        </td>
+        <td>Total:{{build.test_runs.count}}<br/>
+            Completed:{{build.test_runs_completed}}<br/>
+            Incomplete:{{build.test_runs_incomplete}}
+        </td>
+        <td>Total: {{build.test_summary.tests_total}}<br/>
+            Passed: {{build.test_summary.tests_pass}}<br/>
+            Failed: {{build.test_summary.tests_fail}}<br/>
+            Skipped: {{build.test_summary.tests_skip}}
```

Comment:
I think we could have one column for each of these numbers. This would make the data a lot easier to read.